docs/edgeql/ddl/links.rst

```diff
@@ -262,17 +262,16 @@ The following subcommands are allowed in the ``ALTER LINK`` block:
     supertypes.
 
 :eql:synopsis:`SET SINGLE`
-    Change the maximum cardinality of the link set to *one*.  Only
+    Change the link set's maximum cardinality to *one*.  Only
     valid for concrete links.
 
 :eql:synopsis:`SET MULTI`
-    Change the maximum cardinality of the link set to *greater than one*.
-    Only valid for concrete links;
+    Change the link set's maximum cardinality to fit any natural number 
+    ( *greater than or equal to one* ). Only valid for concrete links.
```

Comment:
```suggestion
    Remove the upper limit on the link set's cardinality. Only valid for 
    concrete links.
```

I think this wording is clearer, instead of trying to talk about "greater than one" or "natural numbers".